runtime/doc/lsp.txt

```diff
@@ -61,6 +61,16 @@ options are not restored when the LSP client is stopped or detached.
 - |K| is mapped to |vim.lsp.buf.hover()| unless |'keywordprg'| is customized or
   a custom keymap for `K` exists.
 
+                                                *grr* *gra* *grn* *i_CTRL-S*
+Some keymaps are created unconditionally when Nvim starts:
+- "grn" is mapped in Normal mode to |vim.lsp.buf.rename()|
+- "gra" is mapped in Normal and Visual mode to |vim.lsp.buf.code_action()|
+- "grr" is mapped in Normal mode to |vim.lsp.buf.references()|
+- CTRL-S is mapped in Insert mode to |vim.lsp.buf.signature_help()|
+
+If not wanted, these keymaps can be removed at any time using
+|vim.keymap.del()| or |:unmap|.
```

Comment:
for `gr` we may want to mention `nnoremap <nowait> gr gr`

Reply:
That doesn't seem to work. If I use that mapping then I'm not able to use any of the new defaults. It reverts to the builtin `gr` behavior.

Reply:
If I use `vim.keymap.set('n', 'gr', 'gr', { nowait = true })` _before_ creating the other `gr` mappings, then I can still use the new mappings, but it doesn't seem to behave any differently than it does without the `<nowait>` mapping. So I'm not sure what it's meant to be doing.

Reply:
> That doesn't seem to work. If I use that mapping then I'm not able to use any of the new defaults. It reverts to the builtin `gr` behavior.

That's exactly what is supposed to do: conveniently disable all these mappings.

Reply:
Ah that makes sense, I misunderstood the original suggestion. I’ll include that

Reply:
78f2c0749c9f4b588b4154e585233e6c5aa76e1e